benchexec/tools/witness2test.py

```diff
@@ -111,5 +111,8 @@ def determine_result(self, returncode, returnsignal, output, isTimeout):
                         status = result.RESULT_FALSE_REACH
                 elif 'UNKNOWN' in output:
                     status = result.RESULT_UNKNOWN
+        elif output and \
+             re.match(r'^INVALID WITNESS FILE', output[-1].strip()) is not None:
```

Comment:
Could you please change the indendation to be a multiple of 4 spaces?

And while we are at it,
```
elif (output and
     re.match(...) is not None):
```
is the recommended way to have line breaks in expressions.